data/mappings/keyboard_aliases.hjson

```diff
@@ -407,6 +407,9 @@
     "lfkeyboards/smk65": {
         "target": "lfkeyboards/smk65/revb"
     },
+    "ll3ma/bongopad": {
```

Comment:
Suggestion not correctly applied
```suggestion
    "ll3macorn/bongopad": {
```

Reply:
but once again i dont go by that anymore and ive changed the directory to ll3ma. why would i have to use the old name?

Reply:
The purpose of the change is to maintain backwards compatibility. The format of the file is:

```json
"<from>": {
    "target": "<to>"
}
```

`ll3ma/bongopad` is the new name of the keyboard so it should not be as you have it.

Reply:
ooh i see now my bad